### PR TITLE
add grpc info api

### PR DESCRIPTION
**Pull request by xlgao-zju**
A part of #278 

Signed-off-by: Xianglin Gao <xlgao@zju.edu.cn>

### CODE REVIEW

File: types/types.proto [hunk 1]
```diff
@@ -370,6 +370,24 @@ message ContainerLogsResponse {
   bytes log = 1;
 }
 
+message InfoRequest {
+}
+
+message InfoResponse {
+  string  ID                    = 1;
+  string  Containers            = 2;
```
Comment (laijs) [inline types/types.proto]: its type is int32

**Comment by laijs** [comment]
change the return type of CmdSystemInfo() to be types.InfoResponse for new gRPC api.

and convert it into engine.Env for old api.
// old api: server/router/system/system_routes.go:18
```go
func (s *systemRouter) getInfo(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
        env, err := s.backend.CmdSystemInfo()
        if err != nil {
                return err
        }

        return env.WriteJSON(w, http.StatusOK)
}
```

File: types/types.proto [hunk 2]
```diff
@@ -370,6 +370,24 @@ message ContainerLogsResponse {
...
+  string  Images                = 3;
```
Comment (laijs) [inline types/types.proto]: int32

File: types/types.proto [hunk 3]
```diff
@@ -370,6 +370,24 @@ message ContainerLogsResponse {
...
+  string  Driver                = 4;
+  string  DriverStatus          = 5;
```
Comment (laijs) [inline types/types.proto]: define a grpc struct

File: types/types.proto [hunk 4]
```diff
@@ -370,6 +370,24 @@ message ContainerLogsResponse {
...
+  string  DockerRootDir         = 6;
+  string  IndexServerAddress    = 7;
+  string  ExecutionDriver       = 8;
+  string  MemTotal              = 9;
+  string  Pods                  = 10;
```
Comment (laijs) [inline types/types.proto]: int32

File: types/types.proto [hunk 5]
```diff
@@ -370,6 +370,24 @@ message ContainerLogsResponse {
...
+message InfoResponse {
+  string  ID                    = 1;
+  string  Containers            = 2;
+  string  Images                = 3;
+  string  Driver                = 4;
+  string  DriverStatus          = 5;
+  string  DockerRootDir         = 6;
+  string  IndexServerAddress    = 7;
+  string  ExecutionDriver       = 8;
+  string  MemTotal              = 9;
```
Comment (laijs) [inline types/types.proto]: uint64 ?

**Comment by xlgao-zju** [comment]
OK, I will do it later.

File: types/types.proto (hunk 4 above)
Comment (xlgao-zju) [inline types/types.proto]: this one is int64

**Comment by xlgao-zju** [comment]
@laijs updated

**Comment by feiskyer** [comment]
LGTM

**Pull request merged**